src/features/expense/model/expenseFormStore.ts

```
@@ -0,0 +1,30 @@
+import { create } from 'zustand';
+import { ExpenseFormState, ExpenseFormActions } from './types';
+
+const initialExpenseFormState: ExpenseFormState = {
+  category: '',
+  date: new Date(),
+  memo: '',
+  amount: 0,
+};
+
+export const useExpenseFormStore = create<ExpenseFormState & ExpenseFormActions>(
+  (set) => ({
```

ESLint

[error] Replace `ExpenseFormState·&·ExpenseFormActions>(⏎··` with `⏎··ExpenseFormState·&·ExpenseFormActions⏎>(`

(prettier/prettier)